features/handled.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Handled Exceptions are received
+
+Scenario: Notify is called
+  Given I configure the bugsnag endpoint
+  And I set environment variable "MAZE_API_KEY" to "1234567890"
+  When I run the console app "DotNetCore2Console" with "handled"
+  Then I should receive a request
+  And the request is a valid for the error reporting API
```

Comment:
It might be useful to verify a few more parts of the payload specific to handled errors. E.g., is `unhandled` false, is the correct `severityReason` set, etc

Reply:
I'll add some of those!